README.yaml

```diff
@@ -126,6 +129,60 @@ usage: |-
     ]
   }
   ```
+
+  With [enhanced monitoring](examples/enhanced_monitoring)
+
+  ```hcl
+  # create IAM role for monitoring
+  resource "aws_iam_role" "iam_role" {
+    name               = "rds-${var.cluster-name}"
```

Comment:
1) use snake case for variables `var.cluster_name`.

2) In examples, it's better to use concrete names instead of variables - easier to read, understand and replicate, e.g.
```
name               = "rds-cluster-1-cloudposse.co"
```

3) We usually use `label` to name all resources - it provides consistency and eliminates naming conflicts in many cases.

Reply:
agreed on all points,  that snuck in there from copying my test I was working with. 

though, I'm not clear what you mean by 3. and labels.  can you clarify?  should I change the example somehow?

Reply:
@rexroof 
we use our `label` module https://github.com/cloudposse/terraform-null-label to name all resources
like here 
https://github.com/cloudposse/terraform-aws-rds-cluster/blob/master/main.tf#L1
https://github.com/cloudposse/terraform-aws-rds-cluster/blob/master/main.tf#L14

so in your case:
```hcl
name               = "${module.label.id}"
```
would be enough to name all resources in the examples.

Reply:
does that mean that you want me to add the label module to the example?  that seems complicated for what should be a simple example.

Reply:
agree, not needed for the examples